libshvbroker/src/rpc/masterbrokerconnection.cpp

```diff
@@ -144,6 +144,14 @@ void MasterBrokerConnection::onRpcDataReceived(shv::chainpack::Rpc::ProtocolType
 				m_connectionState.pingRqId = 0;
 				return;
 			}
+			else {
+				cp::RpcValue rpc_val = decodeData(protocol_type, msg_data, 0);
+				if (rpc_val.isValid()) {
+					rpc_val.setMetaData(std::move(md));
```

Comment:
this move will cause that `BrokerApp::instance()->onRpcDataReceived(connectionId(), protocol_type, std::move(md), std::move(msg_data));` will be called with invalid meta-data